docs/porting/nuget-packages.md

```diff
@@ -1,5 +1,36 @@
-# 🔧 Porting NuGet packages
+# Porting NuGet Packages
 
-> **Note**
-> 
-> This topic hasn’t been written yet! You can track the status of this issue through our public GitHub issue tracker. Learn how you can contribute on GitHub.
+This article covers how to get your ported .NET Framework code on NuGet.  Although not much has changed, there are different avenues with which you can package your code.
+
+## Prerequisities
+
+This article assumes that you have already ported your code to .NET Core.  If you have not done that, you can follow [Porting Your Libraries](libraries.md), as it contains relevant information if your code is a library or a framework.
+
+Cross-platform: [.NET CLI](aka.ms/dotnetcoregs)
+
+## Using the Cross-Platform .NET CLI to Generate a NuGet Package
+
+With the .NET CLI, you can build a NuGet package on your OS of choice.
+
+For example, imagine you had a solution with two projects: `Package.Core` and `Package`.  `Package` is where all the public APIs live, and `Package.Core` is where all the executing code is placed.  This package is based on .NET Standard 1.5.
```

Comment:
This is super unclear. You could say "`Package.Core` contains some private implementations".

Also, put a link on Standard 1.5 to the Net Standards page.